docs/src/test/scala/docs/http/scaladsl/server/directives/JsonStreamingFullExamples.scala

```diff
@@ -4,6 +4,7 @@
 
 package docs.http.scaladsl.server.directives
 
+import com.github.ghik.silencer.silent
```

Comment:
Can we have fatal warnings but not for all warning types? It's not that `-Xlint:unused` is useless but it just shouldn't fail the build. ;)